code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/ShumihinSergeyTest.java

```diff
@@ -0,0 +1,36 @@
+package ru.unn.agile.ComplexNumber.Model;
+
+import org.junit.Test;
+import ru.unn.agile.ComplexNumber.model.ComplexNumber;
+import static org.junit.Assert.*;
+
+public class ShumihinSergeyTest {
+    @Test
+    public void canAddComplexNumbers() {
+        ComplexNumber firstNumber = new ComplexNumber(1.0, 0.0);
+        ComplexNumber secondNumber = new ComplexNumber(0.0, -1.0);
+
+        ComplexNumber resultNumber = firstNumber.add(secondNumber);
+
+        assertEquals(new ComplexNumber(1.0, -1.0), resultNumber);
```

Comment:
Magic constants like here are not good. Could you create a dedicated `expected` variables?

Reply:
Not only in this test

Reply:
Making `new ComplexNumber(1.0, -1.0)` a variable won't change much, because this line is not complex to understand. 
If we talking about `1.0` and `-1.0` as magic values, adding variables for this two numbers will just overcomplicate code instead of simplifying. In terms of readability.

Reply:
OK, thanks